From 1a79ffd752b640cb023b6155b6c9b4920e0fb123 Mon Sep 17 00:00:00 2001
From: Kyle Deng <quic_chunkaid@quicinc.com>
Date: Wed, 23 Oct 2024 14:55:52 +0800
Subject: [PATCH 1/3] FROMLIST:dt-bindings: mailbox: qcom,apcs-kpss-global:
 Document the qcs615 APSS

Add compatible for the Qualcomm qcs615 mailbox block.
QCS615 mailbox is compatible with SDM845 use fallback for it.

Signed-off-by: Kyle Deng <quic_chunkaid@quicinc.com>
Link: https://lore.kernel.org/all/20241018073417.2338864-2-quic_chunkaid@quicinc.com/
Upstream-Status: Submitted[https://lore.kernel.org/all/20241018073417.2338864-2-quic_chunkaid@quicinc.com/]
---
 .../devicetree/bindings/mailbox/qcom,apcs-kpss-global.yaml       | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/mailbox/qcom,apcs-kpss-global.yaml b/Documentation/devicetree/bindings/mailbox/qcom,apcs-kpss-global.yaml
index d2e25ff6db7f..970b7a79228e 100644
--- a/Documentation/devicetree/bindings/mailbox/qcom,apcs-kpss-global.yaml
+++ b/Documentation/devicetree/bindings/mailbox/qcom,apcs-kpss-global.yaml
@@ -25,6 +25,7 @@ properties:
           - const: qcom,ipq6018-apcs-apps-global
       - items:
           - enum:
+              - qcom,qcs615-apss-shared
               - qcom,sc7180-apss-shared
               - qcom,sc8180x-apss-shared
               - qcom,sm8150-apss-shared
-- 
2.34.1

